src/org/omegat/core/machinetranslators/DeepLTranslate.java

```diff
@@ -91,7 +90,7 @@ protected String translate(Language sLang, Language tLang, String text) throws E
         String apiKey = getCredential(PROPERTY_API_KEY);
 
         if (apiKey == null || apiKey.isEmpty()) {
-            return OStrings.getString("DEEPL_API_KEY_NOTFOUND");
+            throw new Exception(OStrings.getString("DEEPL_API_KEY_NOTFOUND"));
```

Comment:
This is bad habit to use exception to pass a failure status and error message, but it is convenient to hack it.